cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -634,13 +634,15 @@ recompiled and re-deployed.)
    - Using Yum (on one line)::
 
        $ sudo yum install cdap cdap-gateway \
-             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
```

Comment:
Yes, this is no different from the other HBase version support (cdap-hbase-compat-0.96 gets installed even if you are running on HBase 0.98).  The startup scripts handle figuring out which version of the jars to include, the same as always, and are smart enough to flag the version of HBase 1.0 on CDH separately.